### PR TITLE
w-13808887-rtf-july-general-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by Crispy-Salesforce — DISMISSED** [review]

File: modules/ROOT/pages/deploy-index.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= Deploying Applications and API Proxies to Runtime Fabric
+= “Deploying and Managing Applications to Runtime Fabric
```
Comment (dmerlob) [inline modules/ROOT/pages/deploy-index.adoc]:
```suggestion
= Deploying and Managing Applications to Runtime Fabric
```

File: modules/ROOT/pages/rtf-cluster-config.adoc [hunk 2]
```diff
@@ -4,14 +4,14 @@ To use the complete value of Runtime Fabric, refer to your respective Kubernetes
 
 [%header%autowidth.spread]
 |===
-| Type of configuration | EKS-A | EKS | Appliance
-| Infrastructure | Vsphere/Bare Metal. | AWS | Pre-provisioned hosts.
+| Type of configuration | EKS-A | EKS | Appliance | MuleSoft requirements
+| Infrastructure | Vsphere/Bare Metal. | AWS | Pre-provisioned hosts. | -
```
Comment (dmerlob) [inline modules/ROOT/pages/rtf-cluster-config.adoc]:
```suggestion
| Infrastructure | Vsphere/Bare Metal | AWS | Pre-provisioned hosts | -
```

File: modules/ROOT/pages/rtf-cluster-config.adoc [hunk 3]
```diff
@@ -4,14 +4,14 @@ To use the complete value of Runtime Fabric, refer to your respective Kubernetes
...
-| Load balancer | https://github.com/metallb/metallb[MetalLB^] | ELB | -
-| Monitoring |Grafana/Prometheus | CloudWatch | Grafana/Prometheus
-| Local cluster registry | https://github.com/goharbor/harbor[Harbor^] or any other compliant implementation. | ECR | Docker registry running inside the cluster.
-| Cluster size limits | Kubernetes imposed limits on the cluster size. | 1000 managed nodes. | 21 total (5 control plane nodes + 16 worker nodes.
-| CK8s distro | https://github.com/aws/eks-distro[EKS-D^] | https://github.com/aws/eks-distro[EKS-D^] | Custom distro (planet).
-| Cluster mgmt tool | `eksctl` or `eksctl-anywhere` | `eksctl` | Custom utility (rtfctl)
+ | Nginx ingress | Nginx ingress | Customer-managed component config
+| Load balancer | https://github.com/metallb/metallb[MetalLB^] | ELB | - | Customer-managed component config
+| Monitoring |Grafana/Prometheus | CloudWatch | Grafana/Prometheus | Check xref:limitations-self.adoc[Requirements and Limitations] of Runtime Fabric
+| Local cluster registry | https://github.com/goharbor/harbor[Harbor^] or any other compliant implementation. | ECR | Docker registry running inside the cluster. |
```
Comment (dmerlob) [inline modules/ROOT/pages/rtf-cluster-config.adoc]:
```suggestion
| Local cluster registry | https://github.com/goharbor/harbor[Harbor^] or any other compliant implementation | ECR | Docker registry running inside the cluster | 
```

File: modules/ROOT/pages/rtf-cluster-config.adoc [hunk 4]
```diff
@@ -4,14 +4,14 @@ To use the complete value of Runtime Fabric, refer to your respective Kubernetes
...
-| Local cluster registry | https://github.com/goharbor/harbor[Harbor^] or any other compliant implementation. | ECR | Docker registry running inside the cluster.
-| Cluster size limits | Kubernetes imposed limits on the cluster size. | 1000 managed nodes. | 21 total (5 control plane nodes + 16 worker nodes.
-| CK8s distro | https://github.com/aws/eks-distro[EKS-D^] | https://github.com/aws/eks-distro[EKS-D^] | Custom distro (planet).
-| Cluster mgmt tool | `eksctl` or `eksctl-anywhere` | `eksctl` | Custom utility (rtfctl)
+ | Nginx ingress | Nginx ingress | Customer-managed component config
+| Load balancer | https://github.com/metallb/metallb[MetalLB^] | ELB | - | Customer-managed component config
+| Monitoring |Grafana/Prometheus | CloudWatch | Grafana/Prometheus | Check xref:limitations-self.adoc[Requirements and Limitations] of Runtime Fabric
+| Local cluster registry | https://github.com/goharbor/harbor[Harbor^] or any other compliant implementation. | ECR | Docker registry running inside the cluster. | 
+| Cluster size limits | Kubernetes imposed limits on the cluster size. | 1000 managed nodes. | 21 total (5 control plane nodes + 16 worker nodes. | -
```
Comment (dmerlob) [inline modules/ROOT/pages/rtf-cluster-config.adoc]:
```suggestion
| Cluster size limits | Kubernetes imposed limits on the cluster size. | 1000 managed nodes | 21 total (5 control plane nodes + 16 worker nodes | -
```

File: modules/ROOT/pages/rtf-cluster-config.adoc [hunk 5]
```diff
@@ -4,14 +4,14 @@ To use the complete value of Runtime Fabric, refer to your respective Kubernetes
...
+| CK8s distro | https://github.com/aws/eks-distro[EKS-D^] | https://github.com/aws/eks-distro[EKS-D^] | Custom distro (planet). | Check xref:limitations-self.adoc[Requirements and Limitations] of Runtime Fabric
```
Comment (dmerlob) [inline modules/ROOT/pages/rtf-cluster-config.adoc]:
```suggestion
| CK8s distro | https://github.com/aws/eks-distro[EKS-D^] | https://github.com/aws/eks-distro[EKS-D^] | Custom distro (planet) | Check xref:limitations-self.adoc[Requirements and Limitations] of Runtime Fabric
```

File: modules/ROOT/pages/secure-runtime-fabric.adoc [hunk 6]
```diff
@@ -0,0 +1,8 @@
+= Secure Runtime Fabric
+ 
+ To secure Anypoint Runtime Fabric, review the following documentation:
```
Comment (dmerlob) [inline modules/ROOT/pages/secure-runtime-fabric.adoc]:
```suggestion
 To secure Anypoint Runtime Fabric, see:
```
This is optional, to reduce the number of words.

File: modules/ROOT/pages/upgrade-openshift.adoc [hunk 7]
```diff
@@ -2,10 +2,12 @@
 
 Red Hat OpenShift console allows you to run upgrades automatically (default) or manually. You can change this setting in the *Subscription* tab in the Runtime Fabric Operator. 
 
+[Note]
+Starting Runtime Fabric agent version 2.4.0 for any new Mule app deployment, you must use a runtime image built after or on Aug 8, 2023.
```
Comment (dmerlob) [inline modules/ROOT/pages/upgrade-openshift.adoc]:
```suggestion
Starting with Runtime Fabric agent version 2.4.0, for any new Mule app deployment, you must use a Mule runtime image built on Aug 8, 2023, or later.
```

**Review by dmerlob — DISMISSED** [review]

**Review by dmerlob — APPROVED** [review]

**Pull request merged**